analyses/manuscript_figures.py

```diff
@@ -71,9 +71,41 @@ def format_responses(responses):
                                     stim_param=[responses['stim_param'][trial]]))
     return response
 
-def get_response(expt, pre, post, type='pulse'):
+def get_response(expt, pre, post, analysis_type = 'pulse'):
+    """Get the a) deconvolved and "aligned to spike" responses from 
+    the DynamicsAnalyzer and b) the cross talk artifact from a specified 
+    experiment. 
+    
+    Parameters
+    ----------
+    expt: multipatch_analysis.experiment.Experiment class
+        Contains information about and data from an experiment.
+    pre: integer
+        Refers to the assigned number of the pre-synaptic neuron  
```

Comment:
ID of the presynaptic cell (`Cell.cell_id`)

Reply:
No, I take it back--it's the ID of the presynaptic electrode (`Electrode.electrode_id`)

Reply:
They are always the same value (so far), but it's good to be precise.